src/services/audioApi.js

```
@@ -1,61 +1,41 @@
-//   return {
-//     baseUrl: REMOTE_AUDIO_API_URL,
-//     headers: {
-//       Authorization: `Basic ${credentials}`,
-//       "Content-Type": "application/json",
-//     },
-//   };
-// };
+const getApiConfig = () => {
+  const { BASIC_AUTH_USERNAME, BASIC_AUTH_PASSWORD, REMOTE_AUDIO_API_URL } = constant;
+  const credentials = btoa(`${BASIC_AUTH_USERNAME}:${BASIC_AUTH_PASSWORD}`);
+  return {
+    baseUrl: REMOTE_AUDIO_API_URL,
+    headers: {
+      Authorization: `Basic ${credentials}`,
+      "Content-Type": "application/json",
+    },
+  };
+};
 
 // Generic API request function
-// const makeApiRequest = async (endpoint, options = {}) => {
-//   try {
-//     const config = getApiConfig();
-//     const fullUrl = `${config.baseUrl}${endpoint}`;
-
-//     console.log("🔗 Making API request to:", fullUrl);
-//     console.log("📱 Platform:", require("react-native").Platform.OS);
-
-//     const response = await fetch(fullUrl, {
-//       method: "GET",
-//       headers: config.headers,
-//       timeout: 15000, // 15 second timeout for slow networks
-//       ...options,
-//     });
-
-//     console.log("📊 Response status:", response.status);
+const makeApiRequest = async (endpoint, options = {}) => {
+  try {
+    const config = getApiConfig();
+    const fullUrl = `${config.baseUrl}${endpoint}`;
 
-//     if (!response.ok) {
-//       console.error(`❌ API Error: ${response.status} - ${response.statusText}`);
-//       return null;
-//     }
+    const response = await fetch(fullUrl, {
+      method: "GET",
+      headers: config.headers,
+      timeout: 15000, // 15 second timeout for slow networks
+      ...options,
+    });
 
-//     const data = await response.json();
-//     console.log("✅ API Response:", data);
-//     return data;
-//   } catch (error) {
-//     console.error("💥 API Request Error:", error.message);
-//     console.error("🔍 Error type:", error.name);
+    if (!response.ok) {
+      return null;
```

Another silently failing request, let me know if this was done intentionally.